rita_client/src/dashboard/interfaces.rs

```diff
@@ -161,13 +165,14 @@ fn set_interface_mode(iface_name: &str, mode: InterfaceMode) -> Result<(), RitaC
         return Err(RitaClientError::InterfaceModeError(
             "Attempted to configure non-existant or unavailable interface!".to_string(),
         ));
-    } else if target_mode == InterfaceMode::Wan {
+    } else if target_mode == InterfaceMode::Wan || target_mode == InterfaceMode::LTE {
         // we can only have one WAN interface, check for others
-        // StaticWAN entires are not identified seperately but if they ever are
```

Comment:
I took a quick pass at StaticWan but it behaves differently due to being a struct enum type.  It would be nice to check, but it's not essential.

Reply:
so this makes wan and LTE exclusive? is that actually required? Since we don't have external nic or any gateway stuff I don't think so.

Reply:
I began with the assumption they would be exclusive, but as I got into it, I think you're right that they could be compatible.

Reply:
Removed the exclusion check